tests/end_to_end/CairoPrecompiles/test_call_cairo_precompile.py

```diff
@@ -105,17 +106,20 @@ async def test_should_fail_when_called_with_callcode(
             ):
                 await cairo_counter_caller.incrementCairoCounterCallcode()
 
+    @pytest.mark.skipif(
```

Comment:
q: why not xfail here? so that once done, this fails and we don't forget it

Reply:
skipif because we skip some test for some networks (e.g. if the network is not up to date, or some other conditions like for messaging)